packages/@lwc/integration-karma/test/component/refs/x/overwrite/overwrite.js

```diff
@@ -0,0 +1,10 @@
+import { LightningElement, api } from 'lwc';
+
+export default class extends LightningElement {
+    refs = 'yolo';
```

Comment:
Nice find! You're right; it does throw:

     TypeError: Cannot set property refs of [object Object] which has only a getter

This actually changes the observable change analysis. If a customer tries to set `this.refs` as an expando, then it will throw whereas it wouldn't before.  So I changed it so that setting `refs` works as an expando (see new tests).